### PR TITLE
COMMERCE-5036 Dont use deafultCompanyId

### PR DESCRIPTION
**Pull request by dsitu**

### CODE REVIEW

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/upgrade/v2_5_0/FriendlyURLEntryUpgradeProcess.java [hunk 1]
```diff
@@ -79,8 +78,7 @@ public void doUpgrade() throws Exception {
 				long classPK = rs.getLong("classPK");
 				long companyId = rs.getLong("companyId");
 				Date date = new Date(System.currentTimeMillis());
-				Group group = _groupLocalService.getCompanyGroup(
-					PortalUtil.getDefaultCompanyId());
+				Group group = _groupLocalService.getCompanyGroup(companyId);
```
Comment (dsitu) [inline modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/upgrade/v2_5_0/FriendlyURLEntryUpgradeProcess.java:81]: @riccardo-alberti I included the upgrade process change here.

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:sf

**Comment by marco-leo** [comment]
ci:forward

**Pull request closed without merging**